### PR TITLE
Add additional type tag for Bind metrics

### LINKED ISSUE #13539: BIND statistics in external datastore (InfluxDB)
### The problem

BIND statistics are stored in files like `app-bind-57.rrd` (incoming queries) and `app-bind-57-outgoing.rrd` (outgoing queries). 
Each of the mentioned RRD files contains metrics called "ANY", "A", "AAAA", "PTR", etc. When the data points are exported to InfluxDB, there is no way to differentiate the same-named metrics. We have multiple "A" points and multiple "AAAA" points in each time bucket, and there's no way to determine if they belong to the incoming queries or outgoing queries data set.

### Output of ./validate.php

```txt
====================================
Component | Version
--------- | -------
LibreNMS  | 21.11.0
DB Schema | 2021_10_20_224207_increase_length_of_attrib_type_column (224)
PHP       | 7.4.3
Python    | 3.8.10
MySQL     | 10.3.31-MariaDB-0ubuntu0.20.04.1
RRDTool   | 1.7.2
SNMP      | NET-SNMP 5.8
====================================

[OK]    Composer Version: 2.1.12
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database schema correct
```

### What was the last working version of LibreNMS?

21.11.0

### Anything in the logs that might be useful for us?

_No response_

### PR DESCRIPTION
**Pull request by murrant**
Fixes data getting clobbered when exported to external TSDB.
Fixes #13539

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/applications/bind.inc.php [hunk 1]
```diff
@@ -127,7 +127,7 @@
 ];
 $metrics['incoming'] = $fields;
 
-$tags = ['name' => $name, 'app_id' => $app_id, 'rrd_def' => $rrd_def, 'rrd_name' => $rrd_name];
+$tags = ['name' => $name, 'app_id' => $app_id, 'client' => 'incoming_extended', 'rrd_def' => $rrd_def, 'rrd_name' => $rrd_name];
```
Comment (mpikzink) [inline includes/polling/applications/bind.inc.php:130]: client > type ?

Comment (murrant) [reply includes/polling/applications/bind.inc.php:130]: @mpikzink for some reason the client queries are split into two data groups...  Take a look at the keys in the two, there is no collision.

Comment (murrant) [reply includes/polling/applications/bind.inc.php:130]: OH, lol, I see i now, thanks.

Comment (bakerds) [reply includes/polling/applications/bind.inc.php]: It is odd that incoming queries are split into two groups, but outgoing are not. I don't think there's really a good reason to tag them differently in TSDBs, other than to make them match the RRD (?)

**Pull request merged**